Documentation/client.md

```diff
@@ -27,7 +27,7 @@ If you don't yet have a Rook cluster running, refer to our [Quickstart Guides](R
 4. Cleanup
 
     ```bash
-    sudo rook block unmount --path /tmp/rook-volume
+    sudo rook block unmap --mount /tmp/rook-volume
```

Comment:
I choose --mount because it specified a mount point.  If you use the other option (--device) we still unmount.  In other words unmap always unmaps an rbd device. You can specify the device with either --device (device name) or --mount (mount point).